cmd/errors/errors.go

```diff
@@ -95,4 +96,7 @@ var (
 
 	// Error/Flag to detect when a user has requested early termination
 	ErrTerminatedByUser = errors.New("terminated by user request")
+
+	ErrIndexTooOld = errors.New("public index too old")
```

Comment:
Is the public index synonym for "index.pidx", maybe we just use the filename everywhere?